Source/Redemption/Public/HealthComponent.h

```diff
@@ -20,37 +20,33 @@ UCLASS( ClassGroup=(Custom), meta=(BlueprintSpawnableComponent) )
 class REDEMPTION_API UHealthComponent : public UActorComponent
 {
 	GENERATED_BODY()
-	DECLARE_DYNAMIC_MULTICAST_DELEGATE(FOnHealthLowDelegate);
```

Comment:
This is what you said that someone created this delegate and no one used it right?

Reply:
yes